shared/common-adapters/icon.native.js

```diff
@@ -31,17 +32,26 @@ export default class Icon extends Component {
     delete containerProps.width
     delete containerProps.height
 
-    return (
-      <TouchableHighlight
-        activeOpacity={0.8}
-        underlayColor={globalColors.white}
-        onPress={this.props.onClick}
-        style={containerProps} >
-        {fontIcons[iconType]
-          ? <Text style={{color, fontFamily: 'kb', ...fontSize}}>{fontIcons[iconType]}</Text>
-          : <Image source={images[this.props.type]} style={{resizeMode: 'contain', ...width, ...height}}/>
-        }
-      </TouchableHighlight>
-    )
+    const icon = fontIcons[iconType]
+      ? <Text style={{color, fontFamily: 'kb', ...fontSize}}>{fontIcons[iconType]}</Text>
+      : <Image source={images[this.props.type]} style={{resizeMode: 'contain', ...width, ...height}}/>
+
+    if (this.props.onClick) {
```

Comment:
Where do you mean to set disabled? this is an icon, it has no disabled attribute: https://github.com/keybase/client/blob/master/shared/common-adapters/icon.js.flow